<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelDocumentInfo</class>
 <widget class="QFrame" name="PanelDocumentInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="uuidLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uuid">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>标题</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="title">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="commentLabel">
     <property name="text">
      <string>注释</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="comment">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="creationToolLabel">
     <property name="text">
      <string>创建软件</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="creationTool">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="creationToolVersionLabel">
     <property name="text">
      <string>软件版本</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="creationToolVersion">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="creationPathLabel">
     <property name="text">
      <string>创建路径</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="creationPath">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="creationHostLabel">
     <property name="text">
      <string>主机名</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="creationHost">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="creationDomainLabel">
     <property name="text">
      <string>所在域名</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="creationDomain">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="partTypeLabel">
     <property name="text">
      <string>工件类型</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="partType">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="partModelLabel">
     <property name="text">
      <string>工件型号</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="partModel">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="partIdentifierLabel">
     <property name="text">
      <string>工件识别码</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="creationTimeLabel">
     <property name="text">
      <string>首次创建时间</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="aquisitionStartLabel">
     <property name="text">
      <string>扫描开始时间</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="aquisitionEndLabel">
     <property name="text">
      <string>扫描结束时间</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="partIdentifier">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="creationTime">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="aquisitionStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="aquisitionEnd">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
